### DISCUSSION TITLE
Chia GUI to view plotting while using swar to plot?

### DISCUSSION THREAD

**Discussion by revlisoft** [swar/Swar-Chia-Plot-Manager#463] [Q&A]
Will plotting done via swar be visible in the Chia GUI?

**Comment by ghost** [comment]
I'm having the same curious questions, I ended up using the gui installer, and setup my .yaml file with the /home/swar/chia-blockchain/venv/bin/chia directory, and they did not show up in the GUI, I could see the 2 test ones I was running in the GUI in swar, but even after I had plots finished via swar, they did not show up in the GUI. I will have to restart the GUI when I get home and see if that picks everything up.

**Comment by farkeytron** [comment]
revlisof: No. Only the finished plots show up in the GUI. The in-progress work done by Swar is only shown in "manager.py view"

revlisoft and default8080: It takes some time for the GUI to show the finished plots... they will show up eventually... maybe up to an hour or two after Swar moves the plot to the final directory.

**Reply by ghost** [reply]
Cool thank you for your info I do appreciate it.
